obspy/io/sac/core.py

```diff
@@ -381,7 +381,12 @@ def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
     :return: A ObsPy Stream object.
```

Comment:
I have second thoughts here. I don't think we want to make "encoding" flag a standard parameter nor we want to encourage users to use it. Its existence is due to the fact that bad SAC files are created when people passing files from one system to another. When they are facing this encoding problem like I have (i.e. someone maybe read the file in Windows system and didn't encode it correctly before writing it to SAC. That's why I have a lot of characters in CP 1252), we have a backup plan to guide them using an "encoding" flag by throwing a user warning. I did document the "encoding" parameter in `obspy.io.sac.sactrace` which I regard as the lowest level that we can hide its existence. 

```
 :param encoding: By default, ASCII encoding is used. ASCII-characters
            encoded in a different encoding scheme will be converted to ASCII
            while other special characters will be replaced by '?'.
 :type encoding: str
```

If you guys think it is better to have it as a standard parameter, I have no problem with it. But it seems to be unnecessary changes to the documents and Obspy functionality that most people will not use nor they want to konw about.